pkg/app/carbonapi/app.go

```diff
@@ -45,6 +45,12 @@ type App struct {
 
 	backend backend.Backend
 
+	// During processing we use two independent queues that share a semaphore to prevent stampeding.
+	// fastQ includes regular requests
+	fastQ chan *renderReq
+	// slowQ contains large requests that could fill the queue and create a stampede.
+	slowQ chan *renderReq
```

Comment:
I think the naming can be improved.
slowQ makes me think of a queue in which requests are slow (like because they are big), or a queue which is read from less frequently.
But as I understood, slowQ is where a huge number of render requests which are sent to. And this huge number of render request are there because of glob breaking logic.
Surely, I don't see this as a blocker, but still I think the naming might cause some misunderstandings.

Reply:
I get your point. I'd do the same 3 months ago.

However, the names must convey the semantics, not the processing speed. I think this is a better approach. I learned this from the Go runtime code where slow branches are named so not because they're slow per se, but semantically. This is a recent learning for me.